src/commcare_cloud/ansible/roles/kafka/files/kafka-reassign-tool.rb

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/ruby
+
+# CodeBase :- https://github.com/dimas/kafka-reassign-tool
```

Comment:
Can we not just download this from github instead of copying into our repo?

Reply:
Updated for download instead.